xml/System.Numerics/Complex.xml

```diff
@@ -2949,32 +3129,61 @@ This API is not CLS-compliant.</summary>
-  
- [!code-csharp[System.Numerics.Complex.Multiply#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.numerics.complex.multiply/cs/multiply2.cs#2)]
- [!code-vb[System.Numerics.Complex.Multiply#2](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.numerics.complex.multiply/vb/multiply2.vb#2)]  
-  
- The multiplication of a complex number, a + bi, and a second complex number, c + di, takes the following form:  
-  
- (ac - bd) + (ad + bc)i  
-  
- If the multiplication results in an overflow in either the real or imaginary component, the value of that component is either <xref:System.Double.PositiveInfinity?displayProperty=nameWithType> or <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>.  
-  
- Languages that do not support custom operators can call the <xref:System.Numerics.Complex.Multiply%2A> method instead.  
-  
- The equivalent method for this operator is <xref:System.Numerics.Complex.Multiply%28System.Numerics.Complex%2CSystem.Numerics.Complex%29?displayProperty=nameWithType>]]></format>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The multiplication of a complex number, a + bi, and a second complex number, c + di, takes the following form:
+
+(ac - bd) + (ad + bc)i
+
+If the multiplication results in an overflow in either the real or imaginary component, the value of that component is either <xref:System.Double.PositiveInfinity?displayProperty=nameWithType> or <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>.
+
+Languages that do not support custom operators can call the <xref:System.Numerics.Complex.Multiply%28System.Numerics.Complex%2CSystem.Numerics.Complex%29?displayProperty=nameWithType> equivalent method instead.
+
+          ]]></format>
         </remarks>
         <altmember cref="M:System.Numerics.Complex.Multiply(System.Numerics.Complex,System.Numerics.Complex)" />
       </Docs>
     </Member>
+    <MemberGroup MemberName="op_Subtraction">
+      <Docs>
+        <summary>Subtracts a specified number from another specified number, where at least one of them is a complex number, and the other could be a double-precision real number.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The <xref:System.Numerics.Complex.op_Subtraction%2A> operators allow performing subtraction operations that involve complex numbers. They enable code such as the following:
+
+[!code-csharp[System.Numerics.Complex.Subtract#2](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.numerics.complex.subtract/cs/subtract2.cs#2)]
+[!code-vb[System.Numerics.Complex.Subtract#2](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.numerics.complex.subtract/vb/subtract2.vb#2)]
+
+The subtraction of a complex number, c + di, from another complex number, a + bi, takes the following form:
```

Comment:
The same comment applies here:
- Subtraction with the individual overloads should be removed from here, since it's repeated for the individual overload.
- The overflow and custom operators sentences can remain here and be removed in the individual overloads.